ui/src/app/pages/workspace/workspace-list.spec.tsx

```diff
@@ -86,22 +89,6 @@ describe('WorkspaceList', () => {
     expect(getCardNames(wrapper)).toEqual(workspaceStubs.map((w) => w.name));
   });
 
-  it('navigates when clicking on the workspace name', async () => {
```

Comment:
I tried for about 30 minutes or so and couldn't get the link clicking validation to work with the memory router. I don't know that this is the critical place to be testing this, anyways.